deepchem/models/tensorgraph/tensor_graph.py

```diff
@@ -279,11 +279,11 @@ def default_generator(self,
           deterministic=deterministic,
           pad_batches=pad_batches):
         feed_dict = dict()
-        if len(self.labels) == 1 and y_b is not None and not predict:
+        if len(self.labels) == 1 and y_b is not None:
```

Comment:
So I initially added the predict flag in April.  
Back then we did not allow arbitrary outputs of predict.

We already have a None check for the generator.  Is there any reason besides performance to not always pass in the tensor?  If a user is a power user and wants performance they can create a dataset without None values